### ISSUE TITLE
docs: operation support matrix downloaded csv is missing header row

### ISSUE THREAD

**Issue by seb-afk** [ibis-project/ibis#8390]
### Please describe the issue

The header seems to be missing in the operation support matrix download.

![image](https://github.com/ibis-project/ibis/assets/9196354/045b4689-26c0-491c-b1d1-a08477a90044)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cpcloud** [comment]
@seb-afk Thanks for the report!

We'll look into this and see if there's something we can do about it. We're a bit at the mercy of `itables` here, but hopefully there's an option we can set to enable the header.

**Comment by seb-afk** [comment]
Sounds good! Not really a problem since one can just copy-paste them from the website I just thought it is worth mentioning :)

**Comment by cpcloud** [comment]
Much appreciated!

**Comment by cpcloud** [comment]
@mwouts Is there a way to include the header when copying, downloading csv/excel?

**Comment by mwouts** [comment]
> @mwouts Is there a way to include the header when copying, downloading csv/excel?

Oh that's interesting. Note that `itables==1.7.0` does not support buttons. I would like to add them in `2.0.0`, which is currently a work in progress. But anyway it's great to see the buttons work already in Quarto.

I think the missing header might comes from the fact that the header is passed as HTML while the core of the table is passed as a JSON data. Would you like to try passing the whole table as HTML with the [`use_to_html` option](https://github.com/mwouts/itables/blob/main/itables/javascript.py#L287)? Also, it's worth checking that there is no [`header=False`](https://datatables.net/forums/discussion/58177/not-include-header-when-export-to-excel) option passed to datatables.net.

**Comment by cpcloud** [comment]
Thanks for the reply! Lemme fiddle with those bits and see if anything shakes out.

**Comment by cpcloud** [comment]
I tried passing:

```
    buttons=[{ "extend": "copy", "header": True }, {"extend":"excel","header":True}, {"extend": "csv", "header":True}],
```

to `show` (based on the example link)

and none of the buttons produced headers 😞 

`use_to_html` has a couple of issues, but the main one is that it removes the buttons entirely.

**Comment by cpcloud** [comment]
Closing this out for now. If someone wants to figure out how to get the header into the copy/download buttons a PR would be more than welcome!

**Comment by mwouts** [comment]
Hey @seb-afk , @cpcloud , the buttons will be officially supported in [`itables==2.0.0`](https://github.com/mwouts/itables/issues/238). I've not seen any issue like the one reported here, but I'd be curious to try with your dataframe, maybe it has a specific shape that causes this?

**Comment by cpcloud** [comment]
This appears to be fixed with `itables==2.0` 🎉 

![image](https://github.com/ibis-project/ibis/assets/417981/67290bd6-2bba-4240-bf55-9c22fda8cc52)

**Comment by cpcloud** [comment]
The first row of the downloaded file looks like this:

```
❯ head -1 ~/downloads/Ibis\ –\ Operation\ support\ matrix\ \(2\).csv
"","Backend","bigquery","clickhouse","dask","datafusion","druid","duckdb","exasol","flink","impala","mssql","mysql","oracle","pandas","polars","postgres","pyspark","risingwave","snowflake","sqlite","trino"
```
